### PR TITLE
taxonomy: packaging translations

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: packaging translations

### CODE REVIEW

**Review by alexgarel — APPROVED** [review]
LGTM

**Comment by alexgarel** [comment]
@teolemon there are errors building taxonomies:
```
ERROR - es:caja already is associated to en:box (packaging_shapes) - es:caja cannot be mapped to entry en:case
ERROR - hu:evőeszközök already is associated to en:eating-utensils (packaging_shapes) - hu:evőeszközök cannot be mapped to entry en:cutlery
ERROR - nl:bestek already is associated to en:eating-utensils (packaging_shapes) - nl:bestek cannot be mapped to entry en:cutlery
ERROR - pt:talheres already is associated to en:eating-utensils (packaging_shapes) - pt:talheres cannot be mapped to entry en:cutlery
```

File: taxonomies/packaging_shapes.txt [hunk 1]
```diff
@@ -744,6 +772,12 @@ pt:Talheres, talher
 en:Cutlery
 fr:Couverts, couvert
 pl:Sztućce
+bg:Съдове за хранене
+da:Bestik
+de:Besteck
+hu:Evőeszközök
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:778]:
```suggestion
```

File: taxonomies/packaging_shapes.txt [hunk 2]
```diff
@@ -744,6 +772,12 @@ pt:Talheres, talher
...
+nl:Bestek
+pt:Talheres
```
Comment (teolemon) [inline taxonomies/packaging_shapes.txt:780]:
```suggestion
```

**Pull request merged**